Makefile

```diff
@@ -42,8 +44,8 @@ build-images: build ## Build the hbase, population, exporter images
 		docker-compose build hbase hbase-populate hbase-to-mongo-export-file hbase-to-mongo-export-directory hbase-to-mongo-export-s3 hbase-to-mongo-export-itest; \
 	}
 
-.PHONY: up
-up: build-images ## Bring up hbase, population, and sample exporter services
+.PHONY: up-all
+up-all: ## Bring up hbase, population, and sample exporter services
```

Comment:
Would you not want up to build the images still? Why would you ever build and then up separately?

Reply:
Because it's quicker, it gives you options